examples/c++-api/Makefile

```diff
@@ -1,10 +1,22 @@
-all: count-demo
+CFLAGS=--std=c++11 \
+       -I ../../lib \
+	   -I ../../third-party/smhasher \
+	   -I ../../third-party/seqan/core/include/
```

Comment:
Maybe, but not a priority here. These examples are provided by way of exposition, and even though I'm adding them to the build here the primary purpose is not for the programs to be run but for the code to be digested by a programmer who wants to use khmer.